testing/Overview.test.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import {mount, shallow, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import ItemDescription from '../client/src/Components/itemDescription.jsx';
+import ShippingInformation from '../client/src/Components/ShippingInformation.jsx'
+import globalFetch from 'jest-fetch-mock';
+import sinon from 'sinon'
+import Overview from '../client/src/Components/Overview.jsx'
+import Cart from '../client/src/AddToCart.jsx'
+import ReactDOM from 'react-dom'
+import ReturnPolicy from '../client/src/Components/ReturnPolicy.jsx';
+import SelectionList from '../client/src/Components/SelectionList.jsx';
+import renderer from 'react-test-renderer';
+
+
+configure({ adapter: new Adapter() });
+
+const mockData = [
+  {
+    "overview":["short id pork picanha nostrud et deserunt mignon meatloaf est meatloaf magna "],
+    "shippingTo":["China"," Wales"," Belgium"," Japan"," France"," America"," Germany"," Canada"," Spain"," Brazil"," South Africa"," Belgium"," India"," Mexico"," Argentina"," Ethiopia"," Jamaica"," Brazil"," Sweden"," Denmark"," Switzerland"," Ghana"," Bangladesh"," Switzerland"," Nigeria"," Sri Lanka"," Ireland"," Italy"," Niger"," Sudan"," Italy"," Argentina"," Pakistan"," Cuba"," North Korea"],
+    "deliveryDate":["Oct","Oct"],
+    "_id":"5c6511ac8f191456a859ab5f","productId":98,"price":92.65,"description":"duis filet short bacon do aliquip nulla do hock in strip nulla in andouille t-bone ribeye ","options":{"ribs":["Picanha","est","meatloaf","andouille","Andouille","steak","quis","consequat","kevin"],"pig":["boudin","Magna","beef","hock","laborum","Excepteur","mignon","id","Ham","dolore","lorem"]},
+    "quantityInStock":37,
+    "availableToReturn":true,
+    "shippingCosts":2.99,
+    "manufacturingTime":4,
+    "shippingFrom":" France",
+    "__v":0
+  }
+]
+
+
+describe("Overview Component", () => {
+  
+  it('Should exist', () => {
+    const wrapper = shallow(<ItemDescription />);
+    expect(wrapper.exists()).toBe(true)
+  });
+  
+  it('Should render without throwing an error', () => {
+    expect(shallow(<ItemDescription />).find('.description').exists()).toBe(true);
```

Comment:
this test isnt actually doing anything.  Find will return `{}` if it doesn't find anything, so this test will always pass, unless there was an error with mounting, which your first test would have picked up.  I recommend you do something else, like making sure the length of the `.find()` return is `> 0`